docs/learn/expert-guides/deploy-multichain-contracts.md

```diff
@@ -0,0 +1,594 @@
+
+## Contract Creation
+
+:::info
+
+Proceed to the next section if you already have an ABI and bytecode ready for deployment.
+
+:::
+
+Begin by visiting the [Getting Started](../smart-contract-developers/getting-started.md) section of smart contracts for guidance on setting up Hardhat, the development environment where the contract will be created and compiled.
+
+Here is a basic contract example to deploy:
+
+```solidity title="contracts/TargetContract.sol"
+// SPDX-License-Identifier: Apache-2.0
+pragma solidity ^0.8.4;
+
+/**
+ * @dev sample contract to be used for a guide to deploy multichain contracts
+ */
+contract TargetContract {
+
+    uint256 public number;
+
+    constructor(uint256 _number) {
+        number = _number;
+    }
+
+    function setNumber(uint256 _number) public {
+        number = _number;
+    }
+}
+```
+
+Once compiled using Hardhat, navigate to the `artifacts/contracts/TargetContract.sol/` directory. The ABI and bytecode will be available in `TargetContract.json`.
+
+<details>
+    <summary>The ABI and the bytecode of the contract: </summary>
+
+```json title="artifacts/contracts/TargetContract.sol/TargetContract.json"
+{
+  "abi": [
+    {
+      "inputs": [
+        {
+          "internalType": "uint256",
+          "name": "_number",
+          "type": "uint256"
+        }
+      ],
+      "stateMutability": "nonpayable",
+      "type": "constructor"
+    },
+    {
+      "inputs": [],
+      "name": "number",
+      "outputs": [
+        {
+          "internalType": "uint256",
+          "name": "",
+          "type": "uint256"
+        }
+      ],
+      "stateMutability": "view",
+      "type": "function"
+    },
+    {
+      "inputs": [
+        {
+          "internalType": "uint256",
+          "name": "_number",
+          "type": "uint256"
+        }
+      ],
+      "name": "setNumber",
+      "outputs": [],
+      "stateMutability": "nonpayable",
+      "type": "function"
+    }
+  ],
+  "bytecode": "0x608060405234801561001057600080fd5b5060405161010f38038061010f83398101604081905261002f91610037565b600055610050565b60006020828403121561004957600080fd5b5051919050565b60b18061005e6000396000f3fe6080604052348015600f57600080fd5b506004361060325760003560e01c80633fb5c1cb1460375780638381f58a146049575b600080fd5b604760423660046063565b600055565b005b605160005481565b60405190815260200160405180910390f35b600060208284031215607457600080fd5b503591905056fea2646970667358221220958426080fb00e4b3e137866f9d1884559c2a31f2d3ee15cd4f2c8aee4a92e6764736f6c63430008110033"
+}
+```
+
+</details>
+
+This file is essential for the upcoming steps and should be moved to the same directory as `main.js`.
```

Comment:
Suggest marking this sentence in bold, as it is an important step.
Or even making it a separate step with a subheading?

```suggestion
**This file is essential for the upcoming steps and should be moved to the same directory as `main.js`.**
```